packages/11ty/content/essay.md

```diff
@@ -11,15 +11,15 @@ abstract: |
 
 > What do I want to do? ... I know now is the time for picture books. An American city is the best, Pittsburgh better than Washington. I know more about such a place. I would want to visit several besides Pittsburgh before deciding. Something perhaps smaller. Toledo, Ohio, maybe. Then I’m not sure a book of photos should be identified locally. American city is what I'm after.... People, all classes, surrounded by bunches of the new down-and-out. Automobiles and the automobile landscape. Architecture, American urban taste, commerce, small scale, large scale, the city street atmosphere, the street smell, the hateful stuff, women's clubs, fake culture, bad education, religion in decay....[^3]
 
-The two men could find common ground in part due to the widespread influence of the 1929 publication *Middletown: A Study in Modern American Culture* by Robert and Helen Lynd, professors of sociology at Columbia and Sarah Lawrence, respectively. A 550-page field investigation by social anthropologists with subject headings including "What Middletown does to get its living" and "The houses in which Middletown lives," this pioneering project attempted an objective analysis of life in a small Midwestern American city (Muncie, Indiana). The Lynds' study was hailed as a very accessible report that was most appealing because it made "no attempt to prove anything" but simply recorded "what was observed." The Lynds introduced their topic by saying their goal "was to study synchronously the interwoven trends that are the life of a small American city. A typical city, strictly speaking, does not exist, but the city studied was selected a shaving many features in common to a wide group of communities." ({% cite id='Lynd 1929', pageNumber='3' %})
+The two men could find common ground in part due to the widespread influence of the 1929 publication *Middletown: A Study in Modern American Culture* by Robert and Helen Lynd, professors of sociology at Columbia and Sarah Lawrence, respectively. A 550-page field investigation by social anthropologists with subject headings including "What Middletown does to get its living" and "The houses in which Middletown lives," this pioneering project attempted an objective analysis of life in a small Midwestern American city (Muncie, Indiana). The Lynds' study was hailed as a very accessible report that was most appealing because it made "no attempt to prove anything" but simply recorded "what was observed." The Lynds introduced their topic by saying their goal "was to study synchronously the interwoven trends that are the life of a small American city. A typical city, strictly speaking, does not exist, but the city studied was selected a shaving many features in common to a wide group of communities." ({% cite 'Lynd 1929' '3' %})
```

Comment:
```suggestion
The two men could find common ground in part due to the widespread influence of the 1929 publication *Middletown: A Study in Modern American Culture* by Robert and Helen Lynd, professors of sociology at Columbia and Sarah Lawrence, respectively. A 550-page field investigation by social anthropologists with subject headings including "What Middletown does to get its living" and "The houses in which Middletown lives," this pioneering project attempted an objective analysis of life in a small Midwestern American city (Muncie, Indiana). The Lynds' study was hailed as a very accessible report that was most appealing because it made "no attempt to prove anything" but simply recorded "what was observed." The Lynds introduced their topic by saying their goal "was to study synchronously the interwoven trends that are the life of a small American city. A typical city, strictly speaking, does not exist, but the city studied was selected a shaving many features in common to a wide group of communities." ({% cite 'Lynd 1929', '3' %})
```